sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -482,7 +509,7 @@ public void analyzeSentimentStringListWithOptions() {
             "The restaurant had amazing gnocchi."
         );
 
-        textAnalyticsAsyncClient.analyzeSentimentBatch(documents, "en", null).subscribe(
+        textAnalyticsAsyncClient.analyzeSentimentBatch(documents, "en", new TextAnalyticsRequestOptions()).subscribe(
```

Comment:
Did you have to change this because of an error in the compiler or something?

Reply:
After revisited the purpose of this codesnippet, found set the TextAnalyticsRequestOptions to null is wrong.
The purpose of this codesnippet is to show the how to use API with TextAnalyticsRequestOptions.
So I will update it to include statistics info.

Reply:
This happens in other places of your tests... I see how you always replace null for TextAnalyticsRequestOptions .
if you leave it null, will the code compile?
I am asking this because I want to make sure this is not because of the addition of AnalyzeSentimentOptions